src/content/docs/fr/reference/experimental-flags/index.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Configuration des flags expérimentaux
+i18nReady: true
+---
+
+Les fonctionnalités expérimentales ne sont disponibles qu'après l'activation d'un flag dans le fichier de configuration d'Astro.
+
+```js title="astro.config.mjs"
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+    experimental: {
+        // activer les flags expérimentaux
+        // pour tester de nouvelles fonctionnalités
+    },
+});
+```
+
+
+Astro propose des flags expérimentaux afin de donner aux utilisateurs un accès anticipé aux nouvelles fonctionnalités pour qu'ils les testent et fassent part de leurs commentaires.
+
+Ces flags vous permettent de participer au développement des fonctionnalités en signalant les problèmes et en partageant vos opinions. La stabilité de ces fonctionnalités n'est pas garantie et elles peuvent inclure des ruptures, même dans de petites versions de `patch`, alors que la fonctionnalité est activement développée.
+
+Nous recommandons de [mettre à jour Astro](/fr/upgrade-astro/#mise-à-jour-vers-la-dernière-version) fréquemment, et de suivre les notes de version dans le [Astro changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) qui vous informera de tout changement nécessaire au code de votre projet. La documentation sur les fonctionnalités expérimentales sera toujours mise à jour pour la version actuelle uniquement.
```

Comment:
Either `changelog d'Astro` or `journal des modifications d'Astro` but not `Astro changelog`:
```suggestion
Nous recommandons de [mettre à jour Astro](/fr/upgrade-astro/#mise-à-jour-vers-la-dernière-version) fréquemment, et de suivre les notes de version dans le [changelog d'Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md) qui vous informera de tout changement nécessaire au code de votre projet. La documentation sur les fonctionnalités expérimentales sera toujours mise à jour pour la version actuelle uniquement.
```